robolectric-resources/src/main/java/org/robolectric/manifest/AndroidManifest.java

```diff
@@ -599,6 +603,11 @@ public FsFile getAssetsDirectory() {
     return receivers;
   }
 
+  public String getReceiverPermission(final int receiverIndex) {
```

Comment:
I'm not a huge fan of this API, since it requires the caller to know the receiver index. Can't this be removed since the permissions are being saved on `BroadcastReceiverData`?